cmd/config/internal/commands/run_test.go

```diff
@@ -283,6 +285,26 @@ apiVersion: v1
 			args: []string{"run", "dir", "--image", "foo:bar", "--", "a=b", "c", "e=f"},
 			err:  "must have keys and values separated by",
 		},
+		{
+			name: "specify username",
+			args: []string{"run", "dir", "--fn-user", "root", "--image", "foo:bar"},
```

Comment:
Out of curiosity, what happens if --fn-user is passed twice with different values?

Reply:
That's an interesting question. It's handled by cobra library. I did a quick experiment and it looks like the latter one will take effect.